dbms/src/TestUtils/mockExecutor.cpp

```diff
@@ -251,9 +284,30 @@ void MockDAGRequestContext::addMockTable(const MockTableName & name, const MockC
     mock_tables[name.first + "." + name.second] = columns;
 }
 
+void MockDAGRequestContext::addExchangeRelationSchema(String name, const MockColumnInfos & columns)
+{
+    exchange_schemas[name] = columns;
+}
+
+void MockDAGRequestContext::addExchangeRelationSchema(String name, const MockColumnInfoList & columns)
+{
+    std::vector<MockColumnInfo> v_column_info;
```

Comment:
consider pre-allocating the container capacity before the loop?

Reply:
Good idea！